script/cibuild.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -e # halt script on error
+
+bundle exec jekyll build
```

Comment:
Also, I think this might need to be:

`jekyll build --config _config.yml`

@llimllib might be able to confirm